app/src/main/java/com/evercheck/flagly/di/FeatureHandlerModule.kt

```diff
@@ -15,9 +13,4 @@ object FeatureHandlerModule {
     @Provides
     fun provideCoroutineContextProvider() =
         CoroutineContextProvider(Dispatchers.Main, Dispatchers.IO)
-
-    @Singleton
-    @Provides
-    fun providePresenter(featureFlagHandlerPresenter: FeatureFlagHandlerPresenter): FeatureFlagActivityContract.Presenter =
-        featureFlagHandlerPresenter
 }
```

Comment:
as CoroutinesProvider can be an interface, so the Provides would be something like this

```
    @Singleton
    @Provides
    fun provideCoroutineContextProvider(): CoroutineContextProvider {
        return object : CoroutineContextProvider {
            override val mainDispatcher: CoroutineDispatcher
                get() = Dispatchers.Main
            override val backgroundDispatcher: CoroutineDispatcher
                get() = Dispatchers.IO
        }
    }
```